rest/accounts/instance-get-example-1/instance-get-example-1.3.x.js

```diff
@@ -0,0 +1,10 @@
+// Download the Node helper library from twilio.com/docs/node/install
+// These consts are your accountSid and authToken from twilio.com/user/account
```

Comment:
I believe twilio.com/user/account has been updated to https://www.twilio.com/console. It would apply to all files